core/src/main/java/org/radargun/utils/Utils.java

```diff
@@ -442,7 +442,7 @@ public static void createOutputFile(String fileName, String fileContent, boolean
     *           a Map where the public method name is the key, and the String parameter is the value
     * @return the modified Object, or <code>null</code> if the field can't be changed
     */
-   public static Object invokeMethodWithString(Object object, Map<String, String> properties) {
+   public static <T> T invokeMethodWithString(T object, Map<String, String> properties) {
```

Comment:
Is there any reason for returning the object at all? Moreover, I think that the name of the method is rather misleading - it should be invokeMethod*s*WithString.